users/templates/registration/password_reset_email.html

```diff
@@ -0,0 +1,12 @@
+{% autoescape off %}
+To initiate the password reset process for your {{ user.get_username }} Sheet Storm Account,
+click the link below:
+
+{{ protocol }}://127.0.0.1:8000{% url 'password_reset_confirm' uidb64=uid token=token %}
```

Comment:
I know that in the all-auth templates it is done in this way, but it is much better to generate whole url in the backend and pass it to email template. The case is that page might work on a different host that `127.0.0.1:8000` and the email will still contain it. 

Btw. which view is supposed to use this and other `password_reset_*` templates?

Reply:
@rwrzesien I think it's: `password_reset`, `password_reset_done`, `password_reset_confirm`, `password_reset_complete` from `django.contrib.auth.views`